build/Dockerfile.bundle

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=oadp-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=stable
 LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.23.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.28.1
```

Comment:
I would update operator-sdk in a separate PR

Reply:
This stuff gets updated as part of `make bundle` based on the operator-sdk used, so I think this is the right place, since it's an auto-generated file.

Reply:
oh I mean just the operator-sdk version (from `1.23.0` to `1.28.1`)

At least to update version, we need to also update Makefile https://github.com/openshift/oadp-operator/blob/master/Makefile#L303-L310

Reply:
How was the bundle generated ? Not via `make bundle` ?

Reply:
I would suggest running (in the following sequence):
- make generate
- make manifests
- make bundle

Reply:
@shubham-pampattiwar thanks for that, I've used above, however without first `make generate`.  I feel like `docs/developer/install_from_source.md` may need small tweaks.

Reply:
done. not updating this part in this PR.